data/transactions/logic/assembler.go

```diff
@@ -786,60 +778,73 @@ func assembleSubstring(ops *OpStream, spec *OpSpec, args []string) error {
 	return nil
 }
 
-func assembleTxn(ops *OpStream, spec *OpSpec, args []string) error {
-	if len(args) != 1 {
-		return ops.error("txn expects one argument")
-	}
-	fs, ok := txnFieldSpecByName[args[0]]
+func txnFieldImm(name string, expectArray bool, ops *OpStream) (*txnFieldSpec, error) {
+	fs, ok := TxnFieldSpecByName[name]
 	if !ok {
-		return ops.errorf("txn unknown field: %#v", args[0])
+		return nil, fmt.Errorf("unknown field: %#v", name)
```

Comment:
`name` is a string. is there any particular reason to avoid using `%s` here ?

Reply:
To quote the name in the error message